recipes/platon/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -e
+
+# install PLATON via pip
+$PYTHON -m pip install --no-deps --ignore-installed --no-cache-dir .
```

Comment:
This could be moved directly into the meta.yml as build/script if you like.

Reply:
Ok. Good point, now that the necessary database download is passed to the user's responsibility.